charts/kube-master/charts/etcd/templates/deployment.yaml

```diff
@@ -43,11 +43,27 @@ data:
     exec etcd \
       --name=kubernikus \
       --data-dir=/var/lib/etcd/new.etcd \
+{{- if .Values.secure.enabled }}
+      --cert-file=/etc/kubernetes/certs/tls-etcd.pem \
+      --key-file=/etc/kubernetes/certs/tls-etcd-key.pem \
+      --client-cert-auth=true \
+      --trusted-ca-file=/etc/kubernetes/certs/etcd-clients-ca.pem \
+      --peer-cert-file=/etc/kubernetes/certs/tls-etcd.pem \
```

Comment:
Shouldn't this be a certificate signed with the `etcd-peers-ca`?

Reply:
@jknipper @BugRoger has a point here I think. we should use a different CA for peer communication. As we don't use more then one etcd I also think the whole peer configuration can be omitted, no?